src/ui/components/Notice/index.js

```diff
@@ -35,6 +35,9 @@ type Props = {|
   actionTarget?: string,
   actionText?: string,
   actionTo?: string | Object,
+  // This declares that the Notice component will be rendered against
+  // a $grey-20 background.
+  againstGrey20?: boolean,
```

Comment:
I thought about it but the usage is entirely based on avoiding a color collision. If I used some other name like "photon background" then it would require the reader to look up the color of the photon background to comprehend the code.